src/reference/00-Getting-Started/07A-Scopes.md

```diff
@@ -379,104 +379,153 @@ lazy val util = (project in file("util"))
 #### Delegation in detail (.value lookup)
 
 Now that we've covered all the details of scoping, we can explain the `.value`
-lookup in detail. It's ok to skip this section at in the beginning.
+lookup in detail.
 To summarize what we've learned so far:
 
-- A particular scope consists of three scope axes: Subprojects, Dependency configuration, and Tasks.
-  You can think of axes like as digits in a number, or (r, g, b) components in a color.
-- There's a special value `Global` scope (written as `*` in shell) that can be applied to either of the axes.
-- There's a special value `ThisBuild` scope (written as `{.}` in shell) only for Subprojects axis.
-- build.sbt can reference a scoped key using `.in(...)` method.
+- Scopes have three axes: a subproject axis, a dependency configuration axis, and a task axis.
+- There's a special value `Global` (written as `*` in shell) for any of the scope axes.
+- There's a special value `ThisBuild` (written as `{.}` in shell) for the subproject scope axes, only.
+- a key can be scoped using the `in` method.
 
-Now let's suppose we have a build definition as follows:
+Now let's suppose we have the following build definition:
 
 ```scala
 lazy val foo = settingKey[Int]("")
 lazy val bar = settingKey[Int]("")
 
 lazy val projA = (project in file("a"))
   .settings(
-    foo := {
-      (bar in Test).value + 1
-    },
+    foo := (bar in Test).value + 1,
     bar in Compile := 1
   )
 ```
 
-Inside of `foo`'s setting body the dependency a scoped key `(bar in Test)` is declared.
-However, as you see, `bar in Test` is undefined in `projA`.
-sbt is able to still resolve `(bar in Test)` to another scoped key,
-and initialize `foo` as `2`.
-This fallback search path is called *scope delegation*.
+Inside of `foo`'s setting body a dependency on the scoped key `(bar in Test)` is declared.
+However, despite `bar in Test` being undefined in `projA`,
+sbt is still able to resolve `(bar in Test)` to a value
+resulting in `foo` being initialized to `2`.
+This is *scope delegation*.
 
-First rule is that the _precedence_ among the axes is:
-Subprojects, Dependency configuration, and then Tasks.
+##### Rule 1: Scope precedence
 
-- Rule 1A: If a scope has more specific Subproject axis, then it will always have higher precedence over other scopes without.
-- Rule 1B: If a scope has more specific Dependency configuration, then it will have higher precedence over specific task scoping.
+The first rule of scope delegation is that scope axes have the following _precedence_:
+subproject axis, dependency configuration axis, and task axis.
 
-What is the expected `name in projB` value in the following build?
+Let's look at an example. Given the following build definition:
 
 ```scala
-scalaVersion in (ThisBuild, packageBin) := "2.12.2"
+val bippy = settingKey[String]("The biddy identifier")
+
+bippy in Global := "abc"
```

Comment:
I think we should go the other way and drop the `packageBin` in the other option: we're showing how being project-scoped has higher precedence than Global-scoped. We shouldn't involve task-scoping yet, imo.

I wanted to avoid using `ThisBuild` because we've not yet introduced (at this point in the guide) how `ThisBuild` interacts in the precedence rules. So, in order to use `Global` instead, I created a brand new key, so I could use `Global` to set its default, fallback value. I'll add a comment to that effect.